gui-wm/hyprland/hyprland-0.42.0-r1.ebuild

```diff
@@ -32,6 +32,17 @@ HYPRPM_RDEPEND="
 	dev-vcs/git
 	virtual/pkgconfig
 "
+
+XWAYLAND_DEPEND="
+	X? (
+		x11-libs/libxcb:0=
+		x11-base/xwayland
+		x11-libs/xcb-util-errors
+		x11-libs/xcb-util-renderutil
```

Comment:
I don't think `x11-libs/xcb-util-renderutil` is needed. I don't have it even installed on my setup.